### ISSUE TITLE
Please export additional VM metrics

### ISSUE THREAD

**Issue by wwentland** [lovoo/xenstats_exporter#4]
Hi, 

it would be much appreciated if the xenstat_exporter could export additional VM metrics such as, but not limited to, `VM_metrics.get_VCPUs_utilisation`, `VM_metrics.get_VCPUs_CPU`, `VM_metrics.get_memory_actual`, ... as this would allow for monitoring of running guests as well.

Thank you!

**Comment by SqiSch** [comment]
Hei Babilen, 

sorry for the delayed answer. It´s not possible at the moment to get these informations via the xen api. The VM metrics only fetachable via RRD files from the xenmaster. I already tried to get these informations via xenAPI, but i always get zeros. Sorry for that.

**Comment by wwentland** [comment]
Fair enough. Thank you for the reply! I'll grab these metrics from xentop then.
